06-single-hop-udp/06-single-hop-udp.md

```diff
@@ -70,3 +70,38 @@ Sending UDP from one iotlab-m3 node to a non-existent neighbor.
 
 No leaks in the packet buffer (check with `gnrc_pktbuf_cmd`).
 Packet loss is irrelevant for this test.
+
+Task #05 - Empty UDP on native
+==============================
+### Description
+
+Sending UDP between two native nodes.
+* Stack configuration:    IPv6 (default)
+* Count:                  10
+* Interval:               100ms
+* Port:                   1337
+* Payload:                0B
+* Destination Address:    Link local unicast (fe80::.../64)
+
+### Result
+
+<10% packets lost on the receiving node.
```

Comment:
is "<=" vs "<" relevant here? "<10%" with only 10 packets is practically equivalent to "0%".

Reply:
Well true :-)